app/javascript/chat/chat.jsx

```diff
@@ -39,9 +39,8 @@ export default class Chat extends Component {
     const chatChannels = JSON.parse(props.chatChannels);
     const chatOptions = JSON.parse(props.chatOptions);
 
-    this.debouncedChannelFilter = debounce(
+    this.debouncedChannelFilter = debounceAction(
```

Comment:
I think this is going to have `{leading: true}` added even if now we're not using it

Reply:
@rhymes about 10 minutes after I got up from my computer I realized this exact same thing! I clearly was too excited about this refactor and forgot that we don’t want to lead true everywhere 😂 I’ll fix it on Monday :)